parseAPI/src/Parser.C

```diff
@@ -1071,7 +1071,7 @@ Parser::finalize_jump_tables()
         Function* f = *fit;
         for (auto jit = f->getJumpTables().begin(); jit != f->getJumpTables().end(); ++jit) {
             jumpTableStart.insert(jit->second.tableStart);
-            jumpTables.insert(std::make_pair(jit->second.block, &(jit->second)));
+            jumpTables.emplace(std::make_pair(jit->second.block, &(jit->second)));
```

Comment:
With emplacement, you don't need the call to make_pair. You can just pass the constructor arguments directly.

    jumpTables.emplace(jit->second.block, &(jit->second));